gunicorn/arbiter.py

```diff
@@ -565,6 +565,10 @@ def spawn_worker(self):
             self.WORKERS[pid] = worker
             return pid
 
+        # Do not inherit the temporary files of other workers
+        for sibling in self.WORKERS.values():
+            sibling.tmp.close()
```

Comment:
Perhaps a more elegant way would be:

```py
import os
import resource

fd_high = resource.getrlimit(resource.RLIMIT_NOFILE)[0]
os.closerange(3, fd_high)
```

Or we could store FDs of temp files of workers in a list and pass the highest number as `fd_high`.

Reply:
I'm not sure it would be safe to assume that all open FDs are `WorkerTmp` files. We could have any number of log files open at this point, no?

Reply:
You have a point :) So unless we keep tracking file descriptors of those `WorkerTmp` files I don't see a better way to solve this problem either.